examples/fp-calc.rs

```diff
@@ -0,0 +1,31 @@
+//! A simple test program to run ruperf against
+//!
+//! Do some calculations
+//!
+//! usage:
+//!     ruperf stat ./fp-calc
+fn main() {
+    let start: i32 = 100;
+    let end: i32 = 100000000;
```

Comment:
I think it would be nice if these were also optional CLI arguments.

Reply:
I will make them have CLI arguments